domain/robotouille.json

```diff
@@ -64,6 +80,13 @@
                 "0": "{0} is a robot"
             }
         },
+        {
+            "name": "isgirl",
```

Comment:
our customer asset is female but I would prefer to have a gender neutral predicate (`iscustomer`) since there are no plans for gender-specific interactions. easier to code 😎